ui/components/src/LineChart/LineChart.tsx

```diff
@@ -198,7 +213,7 @@ export function LineChart({ height, data, yAxis, unit, grid, legend, onDataZoom,
       }}
       onDoubleClick={handleOnDoubleClick}
     >
-      {showTooltip === true && (
+      {showTooltip === true && (option.tooltip as TooltipComponentOption).showContent === false && (
```

Comment:
I'm going to play around with [echarts.ComposeOption](https://apache.github.io/echarts-handbook/en/basics/import/#creating-an-option-type-in-typescript) later to see if I can make it so the `as TooltipComponentOption` isn't necessary but this looks good for now!